ktor-network/posix/src/io/ktor/network/sockets/TCPServerSocketNative.kt

```diff
@@ -65,28 +64,28 @@ internal class TCPServerSocketNative(
                 }
             }
         }
+        buildOrClose(clientDescriptor) {
```

Comment:
If accept descriptor setup fails the descriptor is now closed which was not done before